chipy_org/apps/job_board/views.py

```diff
@@ -0,0 +1,57 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from django.views.generic import ListView, DetailView
+from django.urls import reverse 
+from chipy_org.apps.job_board.forms import JobPostForm, JobUserForm, JobProfileForm
+from django.contrib.auth.decorators import login_required
+from .models import JobPost
+from django.db.models import Q
+from itertools import chain
+
+@login_required
+def create_job_post(request):
+
+
+    if request.method == 'POST':
```

Comment:
Thanks for the recommendation. I originally started it with trying to implement it with a ListView, but I found it to be limiting. Then I switched to a function based view since I've worked with it before and I can mold it in the future as I anticipate we'll be making changes to these views.

Reply:
I second these recommendations.  Splitting the logic for get vs post into methods is much cleaner, and generally easier to maintain, over time.